jujugui/static/gui/src/app/components/user-profile/entity/entity.js

```diff
@@ -319,6 +333,102 @@ YUI.add('user-profile-entity', function() {
       );
     },
 
+    /**
+      Retrieve metrics from the plans service.
+
+      @method _getMetrics
+      @param {Object} filters Additional filters to add to the metrics query.
+    */
+    _getMetrics: function(filters) {
+      if (this.state.hasMetrics) {
+        return;
+      }
+      // TODO filters such as summary-interval and start/end dates
+      // can be passed in here, but will need UX first.
+      // Makyo - 2017-03-10
+      this.props.getKpiMetrics(
+          // XXX Metrics are not fully implemented in Omnibus yet. Until they
+          // are, a dummy charm is used. The correct implementation is
+          // commented out below, with the dummy charm in its place.
+          // To QA, uncomment the following line and comment out the line
+          // after that.
+          // Makyo - 2017-03-17
+          //'cs:~canonical/jimm-0',
+          this.props.entity.id,
+          filters,
+          (error, charmMetrics) => {
+            if (error) {
+              // TODO When there are designs for showing errors for metrics,
+              // we'll be able to implement them here.
+              // Makyo - 2017-04-13
+              console.error(error);
+              return;
+            }
+            this.setState({
```

Comment:
Not sure it's necessary at all, actually. We only trigger this code if hasMetrics is false, in which case it should already have these values, right?